src/components/AuthCard.js

```diff
@@ -1,14 +1,14 @@
 import React, { useState, useEffect } from "react";
 import { useHistory, useLocation } from "react-router-dom";
-import { makeStyles } from "@material-ui/core/styles";
+import { makeStyles, StylesProvider } from "@material-ui/core/styles";
```

Comment:
`StylesProvider` is an unused import. Please remove it to keep clean code.
```suggestion
import { makeStyles } from "@material-ui/core/styles";
```